### ISSUE TITLE
export materialmodule was not found in angular/material

### ISSUE THREAD

**Issue by Vikram-ksingh** [isaacplmann/ngx-tour#53]
have installed all required modules but getting this error on npm start or build

export materialmodule was not found in angular/material

following is my dependencies list

`"dependencies": {`
    `"@agm/core": "1.0.0-beta.0",`
    `"@angular/animations": "^4.4.4",`
   `"@angular/cdk": "^2.0.0-beta.12",`
   ` "@angular/common": "^4.2.2",`
    `"@angular/compiler": "^4.2.2",`
    `"@angular/core": "^4.2.2",`
    `"@angular/flex-layout": "2.0.0-beta.8",`
    `"@angular/forms": "^4.2.2",`
    `"@angular/http": "^4.4.4",`
    `"@angular/material": "^2.0.0-beta.10",`
    `"@angular/platform-browser": "^4.4.2",`
    `"@angular/platform-browser-dynamic": "^4.4.2",`
    `"@angular/router": "^4.2.2",`
    `"@ng-bootstrap/ng-bootstrap": "^1.0.0-alpha.26",`
    `"@swimlane/ngx-datatable": "9.3.0",`
    `"angular-calendar": "0.17.0",`
    `"angular-sortablejs": "^2.0.6",`
    `"angular-tree-component": "3.7.3",`
    `"chart.js": "2.6.0",`
    `"ckeditor": "4.6.0",`
    `"core-js": "2.4.1",`
    `"css-element-queries": "^0.4.0",`
    `"d3": "^3.5.17",`
    `"dragula": "3.7.2",`
    `"easy-pie-chart": "2.1.7",`
    `"font-awesome": "4.7.0",`
    `"hammerjs": "2.0.8",`
    `"jquery": "2.2.4",`
    `"leaflet": "^1.0.2",`
    `"leaflet-map": "0.2.1",`
    `"moment": "^2.15.0",`
    `"ng2-breadcrumb": "0.5.14",`
    `"ng2-charts": "1.6.0",`
    `"ng2-ckeditor": "1.1.8",`
    `"ng2-device-detector": "^1.0.0",`
    `"ng2-dragula": "1.3.1",`
    `"ng2-file-upload": "1.2.1",`
    `"ng2-translate": "5.0.0",`
    `"ng2-validation": "4.2.0",`
    `"ng2modules-easypiechart": "^0.0.4",`
    `"ngx-color-picker": "^4.0.3",`
    `"ngx-perfect-scrollbar": "^2.0.1",`
    `"ngx-quill": "^1.2.0",`
    `"ngx-tour-md-menu": "^1.0.4",`
    `"ngx-tour-ng-bootstrap": "^1.0.6",`
    `"normalize.css": "^5.0.0",`
    `"nvd3": "^1.8.5",`
    `"quill": "^1.2.6",`
    `"rxjs": "5.4.1",`
    `"screenfull": "^3.2.2",`
    `"sortablejs": "^1.6.0",`
    `"ts-helpers": "^1.1.2",`
    `"withinviewport": "^2.0.0",`
    `"zone.js": "0.8.14"`
  `},`

**Comment by isaacplmann** [comment]
`ngx-tour-md-menu` works with `@angular/material@2.0.0-beta.11` and `@angular/cdk@2.0.0-beta.11`.  I'll get it working on beta 12 soon.

So change these lines:
```
    "@angular/cdk": "2.0.0-beta.11",
    ...
    "@angular/material": "2.0.0-beta.11",
```

**Comment by josephserrano15** [comment]
Thanks for your reply, we are having the same issue. Do you know hen you will get this done?

**Comment by isaacplmann** [comment]
I'd like to get it working this week, but other priorities might push that back.

**Comment by Vikram-ksingh** [comment]
I got this working with "@angular/material": "2.0.0-beta.8" and "@angular/cdk": "2.0.0-beta.8", there were some issues with other modules I had to resolve

**Comment by josephserrano15** [comment]
Thanks for the update @isaacplmann

**Comment by isaacplmann** [comment]
Just released `ngx-tour-md-menu@2.0.0-beta.12` with this fix.
